src/lib/structures/expr.mli

```
@@ -522,6 +522,8 @@ end
     https://smtlib.cs.uiowa.edu/theories-FixedSizeBitVectors.shtml
     https://smtlib.cs.uiowa.edu/logics-all.shtml#QF_BV *)
 module BV : sig
+  val of_Z : size:int -> Z.t -> t
```

We expose this function that raises an assertion if `size <= 0`. We should raise `invalid_arg` instead.